src/measure_theory/function/strongly_measurable/ae_sequence.lean

```diff
@@ -0,0 +1,120 @@
+/-
+Copyright (c) 2023 Alex Kontorovich and Heather Macbeth. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Alex Kontorovich, Heather Macbeth
+-/
+
+import measure_theory.function.strongly_measurable.basic
+
+/-!
+# Sequence of strongly measurable functions associated to a sequence of a.e.-strongly-measurable
+  functions
```

Comment:
```suggestion
# Sequence of strongly measurable functions associated to a sequence of a.e.-strongly-measurable
# functions
```
Right now this renders badly, see https://observablehq.com/@bryangingechen/github-lean-doc-preview?url=%22https%3A%2F%2Fgithub.com%2Fleanprover-community%2Fmathlib%2Fpull%2F18380%22#docs.